src/components/gataca-autoqr/gataca-autoqr.tsx

```diff
@@ -0,0 +1,259 @@
+            createSession={this.createSession || this.config.createSession}
+            successCallback={
+              this.successCallback || this.config.successCallback
+            }
+            errorCallback={this.errorCallback || this.config.errorCallback}
+            qrRole={this.config.qrRole}
+            callbackServer={this.config.callbackServer}
+            sessionTimeout={this.config.sessionTimeout}
+            pollingFrequency={this.config.pollingFrequency}
+            autorefresh={this.config.autorefresh}
+            v2={this.config.v2}
+            qrModalTitle={this.config.qrModalTitle}
+            qrModalDescription={this.config.qrModalDescription}
+            hideBrandTitle={this.config.hideBrandTitle}
+            dynamicLink={this.config.dynamicLink}
+          />
+        );
+      }
+    } else {
+      if (this.config.useWs) {
+        this.qr = (
+          <gataca-qrws
+            wsOnMessage={this.wsOnMessage || this.config.wsOnMessage}
+            wsOnOpen={this.wsOnOpen || this.config.wsOnOpen}
+            successCallback={
+              this.successCallback || this.config.successCallback
+            }
+            errorCallback={this.errorCallback || this.config.errorCallback}
+            qrRole={this.config.qrRole}
+            callbackServer={this.config.callbackServer}
+            sessionTimeout={this.config.sessionTimeout}
+            socketEndpoint={this.config.socketEndpoint}
+            autostart={this.config.autostart}
+            autorefresh={this.config.autorefresh}
+            v2={this.config.v2}
+            qrModalTitle={this.config.qrModalTitle}
+            qrModalDescription={this.config.qrModalDescription}
+            hideBrandTitle={this.config.hideBrandTitle}
+            dynamicLink={this.config.dynamicLink}
+          />
+        );
+      } else {
+        this.qr = (
+          <gataca-qr
+            checkStatus={this.checkStatus || this.config.checkStatus}
+            createSession={this.createSession || this.config.createSession}
+            successCallback={
+              this.successCallback || this.config.successCallback
+            }
+            errorCallback={this.errorCallback || this.config.errorCallback}
+            qrRole={this.config.qrRole}
+            callbackServer={this.config.callbackServer}
+            sessionTimeout={this.config.sessionTimeout}
+            pollingFrequency={this.config.pollingFrequency}
+            autostart={this.config.autostart}
+            autorefresh={this.config.autorefresh}
+            v2={this.config.v2}
+            qrModalTitle={this.config.qrModalTitle}
+            qrModalDescription={this.config.qrModalDescription}
+            hideBrandTitle={this.config.hideBrandTitle}
+            dynamicLink={this.config.dynamicLink}
+          />
+        );
+      }
+    }
+  }
+
+  render() {
+    if (this.loading) {
+      //TODO: Improve loading message
+      return <div>Loading...</div>;
```

Comment:
I know is not finished. This is just a reminder. Apply internationalization

Reply:
It is very complicated for the AutoQR to apply internationalization with configured texts.

For the other QRs, we should do it for the error messages. It's not done yet.